ui/components/app/signature-request-siwe/index.scss

```diff
@@ -52,8 +52,15 @@
       cursor: pointer;
     }
 
-    .check-box {
+    .mm-box {
       color: var(--color-error-default);
     }
+    .mm-checkbox__input:hover {
+      background-color: var(--color-error-default);
+    }
+    .mm-checkbox__input,  .mm-checkbox__input:focus {
+      border-color: var(--color-error-default);
+      background-color: var(--color-background-default);
+    }
```

Comment:
Can we add a custom className here so we aren't using the default one from the component incase it gets changed?